src/interfaces.ts

```diff
@@ -0,0 +1,15 @@
+import { Router } from 'express';
+
+export interface Controller {
+  path?: string;
```

Comment:
Should this be called `basePath` or similar? I expect a controller defines several paths during boot time, all based on a base path.

Also, what is the use case for a Controller to not have this property declared?

Reply:
> Also, what is the use case for a Controller to not have this property declared?

We should specify that `basepath` should always be, so I will change `basepath` to the required field.